projects/igniteui-angular/src/lib/list/list.component.ts

```diff
@@ -32,6 +37,12 @@ export interface IListItemClickEventArgs {
     event: Event;
 }
 
+export interface IListItemPanningEventArgs {
+    item: IgxListItemComponent;
+    direction: IgxListPanState;
+    cancel: boolean;
```

Comment:
We should probably consider renaming the "cancel" property to something like "keepItem"? Because what it does is to tell the list to bring the panned item back if view after the event is fired.

Reply:
Property renamed